### DISCUSSION TITLE
The taskbar flickers when I open other applications

### DISCUSSION THREAD

**Discussion by eduardopetta** [rocksdanister/lively#512] [Q&A]
The taskbar flickers when I open other applications

https://user-images.githubusercontent.com/84684313/119275271-0a047780-bbe2-11eb-98ee-dcf10b817eed.mp4

**Comment by rocksdanister** [comment]
Discussions are not for bug report.

Taskbar is getting refreshed by windows in some instances, this is a known issue for the time being.
